src/Storages/StorageGenerateRandom.cpp

```diff
@@ -361,6 +361,13 @@ ColumnPtr fillColumnWithRandomData(
             column->getData().resize(limit);
             /// NOTE This is slightly incorrect as random UUIDs should have fixed version 4.
             fillBufferWithRandomData(reinterpret_cast<char *>(column->getData().data()), limit, sizeof(UUID), rng);
+            std::ranges::for_each(
```

Comment:
While yes, since this data is just random bits, since we're seeding the RNG and to ensure backwards compatibility with the existing functional tests I needed to introduce the component shuffling. The `transformEndianness` is also incidentally there to fix a separate BE functional test.